recipes/gmgc-mapper/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "GMGC-mapper" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9a22f7b889e82a0270314dbaa73c3d49055d84df51e3aacddb4fd959ba36e459
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - gmgc-mapper=gmgc_mapper.main:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - atomicwrites
+    - biopython
+    - numpy
+    - pip
+    - python
+    - pyyaml
+    - scikit-bio
+    - tqdm
+  run:
+    - atomicwrites
+    - biopython
+    - numpy
+    - python
+    - pyyaml
+    - scikit-bio
+    - tqdm
```

Comment:
Add `prodigal` as a dependency. There are workflows where it's not necessary, but for the main workflow, it is used.